jax/core.py

```diff
@@ -1678,6 +1678,32 @@ def call_impl(f: lu.WrappedFun, *args, **params):
 named_call_p: CallPrimitive = CallPrimitive('named_call')
 named_call_p.def_impl(call_impl)
 
+outfeed_primitives: Set[Primitive] = set()
```

Comment:
I moved this stuff from dispatch to core so that we can access it from partial_eval.